src/Views/ListView/ColumnBrowser/ColumnBrowser.vala

```diff
@@ -209,7 +206,6 @@ public abstract class Noise.ColumnBrowser : Gtk.Grid {
 
         column_chooser_menu.append (column.menu_item);
 
-        column.row_activated.connect (column_row_activated);
```

Comment:
Hm this looks like a feature regression :)

Reply:
Re-introduced this feature